dev/sg/internal/run/bazel_command.go

```diff
@@ -88,6 +89,18 @@ func (bc BazelCommand) GetExecCmd(ctx context.Context) (*exec.Cmd, error) {
 	return exec.CommandContext(ctx, "bash", "-c", fmt.Sprintf("%s\n%s", bc.Config.PreCmd, cmd)), nil
 }
 
+// Overrides the behavior of this command with other command.
```

Comment:
```suggestion
// Merge overrides the behavior of this command with other command.
```

(go docs expects the inline docs to start with the function name, we don't enforce this though)